barpyrus/trayer.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+from barpyrus.widgets import Widget
+from barpyrus.core import EventInput
+from Xlib.display import Display, X
+
+class TrayerWatch(EventInput):
+    def __init__(self, args = { }):
```

Comment:
Mutable default arguments are tricky in Python - all instances of this class would have the same dictionary this way.

See https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Might not matter here as there probably is just one `TrayerWatch`, but I'd still set `args=None` here and guard it with an `if args is not None:` or so below.

Reply:
I wasn't aware of this :-O, thanks! (And I should check my python code...)

Reply:
Oh, that is indeed good to know :o Thanks a lot!